src/core/CMakeLists.txt

```diff
@@ -47,7 +47,8 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 add_library(ov_core_dev INTERFACE)
 add_library(openvino::core::dev ALIAS ov_core_dev)
 
-target_include_directories(ov_core_dev INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/core/dev_api>)
+target_include_directories(ov_core_dev INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/core/dev_api>
+                                       INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/common/transformations/include>)
```

Comment:
we already have this in master:
https://github.com/openvinotoolkit/openvino/blob/6f3b37b2fabf6409beee6e5f1440f4f294163972/src/core/CMakeLists.txt#L50-L54